userspace/libscap/CMakeLists.txt

```diff
@@ -32,36 +31,33 @@ if(CMAKE_SYSTEM_NAME MATCHES "Linux")
 	endif()
 
 	# do not remove this since when WITH_DRIVER is off
-	if(NOT DEFINED DRIVER_NAME)
-		set(DRIVER_NAME "scap")
-	endif()
+	set(DRIVER_NAME "scap" CACHE STRING _)
```

Comment:
Why is this change needed? If an adopter of the libraries wants to change the name for the driver they should be allowed to, don't you think?